lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -296,7 +296,8 @@
     - ' - C(dns_suffix) (list): List of domain suffixes, also known as DNS search path (default: C(domain) parameter).'
     - ' - C(domain) (string): DNS domain name to use.'
     - ' - C(hostname) (string): Computer hostname (default: shorted C(name) parameter). Allowed characters are alphanumeric (uppercase and lowercase)
```

Comment:
Since this is required on 2.8, I guess it's better to remove the (default: ...) here.
```suggestion
    - ' - C(hostname) (string): Computer (short) hostname. Allowed characters are alphanumeric (uppercase and lowercase)
```

Reply:
The code I proposed in my latest comment changes this.